src/main/resources/db/changelog/changes/v1.1.0/sql/entity-type-definitions/insert-organization-definition.sql

```diff
@@ -0,0 +1,134 @@
+INSERT INTO entity_type_definition (id, derived_table_name, definition)
+    VALUES ('837f262e-2073-4a00-8bcc-4e4ce6e669b3','drv_organization_details', '{
+             "id": "837f262e-2073-4a00-8bcc-4e4ce6e669b3",
+             "name":"drv_organization_details",
+             "private" : false,
+             "fromClause" : "src_organizations as org",
+             "columns" : [
+                {
+                  "name": "last_updated",
+                  "dataType": {
+                    "dataType": "dateType"
+                  },
+                  "valueGetter": "org.jsonb->''metadata''->>''updatedDate''",
+                  "visibleByDefault": false
+                },
+                {
+                  "name": "code",
+                  "dataType": {
+                    "dataType": "stringType"
+                  },
+                  "valueGetter": "org.jsonb->>''code''",
+                  "filterValueGetter": "${tenant_id}_mod_organizations_storage.f_unaccent(org.jsonb->>''code''::text)",
+                  "valueFunction": "${tenant_id}mod_organizations_storage.f_unaccent(:value)",
```

Comment:
I thought you had to use XML or similar for `${tenant_id}` to interpolate

Reply:
If Bobby is to be believed (I dunno... he's a pretty sketchy dude... 😉), it works in .sql files 🙂